src/common/libflux/Makefile.am

```diff
@@ -34,7 +34,7 @@ intree_conf_cppflags = \
 	-DINTREE_EXEC_PATH=\"$(abs_top_builddir)/src/cmd:$(abs_top_srcdir)/src/cmd:$(abs_top_builddir)/src/broker\" \
 	-DINTREE_LUA_PATH_ADD=\"$(abs_top_srcdir)/src/bindings/lua/?.lua\" \
 	-DINTREE_LUA_CPATH_ADD=\"$(abs_top_builddir)/src/bindings/lua/?.so\" \
-	-DINTREE_PYTHON_PATH=\"$(abs_top_builddir)/src/bindings/python\" \
+	-DINTREE_PYTHON_PATH=\"$(abs_top_srcdir)/src/bindings/python:$(abs_top_builddir)/src/bindings/python\" \
```

Comment:
Just to confirm, the srcdir is first because of environment using push_front?

Reply:
Right.  This causes it to come out in the opposite order in PYTHON_PATH.